GitAutoDeploy.py

```diff
@@ -435,10 +438,10 @@ def get_config(self):
             if not os.path.isdir(repo_config['path']):
 
                 print "Directory %s not found" % repo_config['path']
-                GitWrapper.clone(url=repo_config['url'], path=repo_config['path'])
+                GitWrapper.clone(url=repo_config['url'], branch=repo_config['branch'] if 'branch' in repo_config else None, path=repo_config['path'])
 
                 if not os.path.isdir(repo_config['path']):
-                    print "Unable to clone repository %s" % repo_config['url']
+                    print "Unable to clone %s branch of repository %s" % (repo_config['branch'] if 'branch' in repo_config else "default", repo_config['url'])
```

Comment:
If no branch is passed to the clone command, the branch, which is defined as default for the repository, will be cloned. In most cases this will be `master`, but other branch names are possible as well.

Therefore I don't think that it is a good idea to assume that `master` will be cloned when no branch name has been passed to `gut clone`.